ERCS/erc-7824.md

```diff
@@ -0,0 +1,332 @@
+---
+eip: 7824
```

Comment:
```suggestion
eip: 7847
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.